alvr/server_openvr/cpp/platform/win32/VideoEncoderNVENC.cpp

```diff
@@ -129,11 +129,33 @@ void VideoEncoderNVENC::Transmit(
     m_NvNecoder->EncodeFrame(vPacket, &picParams);
 
     for (std::vector<uint8_t>& packet : vPacket) {
+        uint8_t* buf = packet.data();
+        int len = (int)packet.size();
+
+        // NVENC's AV1 encoding includes a bunch of IVF wrapping,
+        // so we need to strip it down to just the OBUs
+        if (m_codec == ALVR_CODEC_AV1) {
+            const uint8_t ivf_magic[4] = { 0x44, 0x4B, 0x49, 0x46 };
+            if (len >= 4 && !memcmp(buf, ivf_magic, 4)) {
+                buf += 0x20;
+                len -= 0x20;
+            }
+            if (len <= 0xC) {
+                continue;
+            }
+            buf += 0xC; // skip past the IVF packet size header thing
+            len -= 0xC;
```

Comment:
Use decimal numbers when assigning or comparing len or buf

Reply:
ehh, not my personal taste but done (spend too much time in hex editors I guess)